tests/integration/helpers.py

```diff
@@ -1029,3 +1030,197 @@ async def backup_operations(
             0
         ], "backup wasn't correctly restored: table 'backup_table_3' exists"
     connection.close()
+
+
+async def pitr_backup_operations(
+    ops_test: OpsTest,
+    s3_integrator_app_name: str,
+    tls_certificates_app_name: str,
+    tls_config,
+    tls_channel,
+    credentials,
+    cloud,
+    config,
+) -> None:
```

Comment:
IMHO this should stay in the PITR test as a hanging functions, since it's not really reusable by anything else.

Reply:
I agree. We can move this now to there or later—no need to block the PR.

Reply:
Should be better now, lighter diff too :)